_posts/2018-08-31-grpc-packaging-releases-nightlies.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: gRPC Packaging: Release Process and Nightly Builds
+published: true
+permalink: blog/grpc-packaging-releases-nightlies
+author: Mehrdad Afshari
+author-link: https://mehrdad.afshari.me
+company: Google
+company-link: https://www.google.com
+---
+
+# gRPC Packaging: Release Process and Nightly Builds #
+*Author: Mehrdad Afshari*
+
+gRPC is [developed in the open on GitHub](https://github.com/grpc)
+and the `master` branch undergoes continuous testing to try to ensure the tip of `master` is buildable and passes the tests.  In this post, we discuss the release process of gRPC as an open source project to clarify it for our users,
+and to serve as an example for other open source projects to consider adopting.  This article is written mostly from the perspective of the gRPC Core (C-based) languages.  The release processes of gRPC Java and Go, although similar in spirit, may be slightly different in technicalities.
+
+## Official Releases ##
+
+gRPC follows semantic versioning and we aim to ship a new minor version every six weeks.  While `master` is quite good and heavily tested, and we could almost always simply consider tagging its `HEAD` as a minor release whenever we like to ship one, a more rigorous quality control process is justified for 
+supported gRPC releases, as it is a critical piece of infrastructure for many users.  As a consequence, we schedule a feature freeze every six weeks.
+[gRPC Release Schedule]() highlights the actual dates.
```

Comment:
Should this be a link?

Reply:
Yes, it should—but @hsaliak should create the destination first.